specification/healthcareapis/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for HealthcareApis service.
 title: HealthcareApisManagementClient
 description: Azure Healthcare APIs Client
 openapi-type: arm
-tag: package-2021-01
+tag: package-preview-2021-06
```

Comment:
Reason to change default to preview version? I don't think python depends on default tag?

Reply:
https://github.com/Azure/sdk-release-request/issues/1809
For singleapi, python relies on default tag of `readme.md`